### ISSUE TITLE
Create a job for waiting on Jenkins to complete before starting the prepartion of helm chart

### ISSUE THREAD

**Issue by umamukkara** [mayadata-io/oep-e2e#10]
As per the design, for each merge into any of the Director repositories, Jenkins and GitLab pipelines will start at the same time.

Jenkins pipeline will build the docker image and then creates a new helm file with the latest image tags.

This issue is for writing a pipeline job in GitLab which will wait for the new helm chart to be updated. More details on how the waiting is done will be described below or in the comments section.
